src/core/registration.c

```diff
@@ -243,37 +243,35 @@ MsQuicRegistrationShutdown(
         QUIC_TRACE_API_REGISTRATION_SHUTDOWN,
         Handle);
 
-    if (Handle && Handle->Type == QUIC_HANDLE_TYPE_REGISTRATION) {
```

Comment:
Why are you removing this?

Reply:
Because it is asserted to be true in lines 233-234.

Reply:
They are debug asserts. They don't fire in release builds. We need these checks still.